bin/lib/book.ml

```diff
@@ -138,10 +143,11 @@ let next_chapter_footer next_chapter : Html.item option =
     front page and only chapter pages contain links to a next chapter,
     so these are additional arguments. *)
 let main_template ?(next_chapter_footer=None)
+    ?(page_title=None)
```

Comment:
Change the definition of the optional arguemnt to be:

```
?page_title
```

rather than explicitly providing None as the default value. That way, when you call main_template, you can pass it an ordinary string (rather than a string option), which will simplify the call point.